extensions/gopass/src/details.tsx

```diff
@@ -58,7 +59,13 @@ export default function ({ entry }: { entry: string }): JSX.Element {
               key={index}
               title={humanize(key)}
               subtitle={value}
-              actions={<Actions copy={() => copy(key, value)} paste={() => paste(key, value)}></Actions>}
+              actions={
+                <Actions
+                  copy={() => copy(key, value)}
+                  paste={() => paste(key, value)}
+                  url={isValidUrl(value) ? value : ""}
```

Comment:
Shouldn't this be `undefined` to avoid adding the `{url && <Action.OpenInBrowser url={url} />}` from the function above?
```suggestion
                  url={isValidUrl(value) ? value : undefined}
```

Reply:
Thank you @mattisssa, I have addressed the title case. 

The condition to show `OpenInBrowser` wasn't the best. I made a small refactor to make the code easier to understand.